contrib/devtools/Makefile

```diff
@@ -122,6 +122,14 @@ ifeq (, $(shell which protoc-gen-swagger))
 else
 	@echo "protoc-gen-grpc-gateway already installed; skipping..."
 endif
+ifeq ($(UNAME_S),Linux)
```

Comment:
I think this is trying to install clang-format everytime `proto-tools` is run. I think it'd be nicer to check if the correct version is installed before trying to run `sudo apt-get install clang-format`